docs/guide/packages/lucide-react.md

```diff
@@ -125,43 +126,31 @@ const Icon = ({ name, ...props }: IconProps) => {
       <LucideIcon {...props} />
     </Suspense>
   );
-}
+};
 
-export default Icon
+export default Icon;
 ```
 
 ##### NextJS Example
 
```

Comment:
We should make separate sections for the App router and pages router.

Pages router should use the docs like it was before.

And create a new section specially targeting the App Router.
With this code example:
```tsx
import { LucideProps, icons } from 'lucide-react';

interface IconProps extends LucideProps {
  name: keyof typeof icons;
}

const Icon = ({ name, ...props }: IconProps) => {
  const LucideIcon = icons[name];

  return <LucideIcon {...props} />;
};

export default Icon;
```